asistente_ladm_col/gui/wizards/operation/wiz_create_points_operation.py

```diff
@@ -245,6 +245,10 @@ def finished_dialog(self):
                                                                field_mapping=field_mapping)
 
                 if res_etl_model:
+                    self.app.core.redraw_all_layers_requested.emit()  # Redraw all layers to show imported data
```

Comment:
No need to emit. If you have `self.app` available, you can call directly `self.app.gui.redraw_all_layers()`

Reply:
Thanks, Change done in 48ddb56